README.md

```diff
@@ -76,6 +76,25 @@ app.use('/admin/queues', router)
 
 That's it! Now you can access the `/admin/queues` route and you will be able to monitor everything that is happening in your queues 😁
 
+### Queue options
+1. `readOnlyMode` (default: `false`)
+Makes the UI as read only,hides all queue & jobs related actions
+   ```js
+    const Queue = require('bull')
+    const QueueMQ = require('bullmq')
+    const { setQueues, BullMQAdapter, BullAdapter } = require('bull-board')
+    
+    const someQueue = new Queue()
+    const someOtherQueue = new Queue()
+    const queueMQ = new QueueMQ()
+    
+    setQueues([
+      new BullAdapter(someQueue, { readOnlyMode: true }), // only this queue will be in read only mode
+      new BullAdapter(someOtherQueue),
+      new BullMQAdapter(queueMQ),
```

Comment:
make it clear that this new option also applied to `BullMQAdapter`

Reply:
Any suggestion of how?

Reply:
Something like this would show the full usage.
```ts
    const someQueue = new Queue()
    const readonlyQueue = new Queue()
    const queueMQ = new QueueMQ()
    const readonlyQueueMQ = new QueueMQ()
    
    setQueues([
      new BullAdapter(someQueue),
      new BullAdapter(readonlyQueue, { readOnlyMode: true }), // queue dashboard will be in read only mode
      new BullMQAdapter(queueMQ),
      new BullMQAdapter(readonlyQueueMQ, { readOnlyMode: true}), // queue dashboard will be in read only mode
```